lib/AuroraLip/Texture/Formats/FIPAFTEX.cs

```diff
@@ -0,0 +1,58 @@
+﻿using AuroraLip.Archives.Formats;
+using AuroraLip.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using static AuroraLip.Texture.Formats.TXE;
+using static AuroraLip.Texture.J3D.JUtility;
+
+
+namespace AuroraLip.Texture.Formats
+{
+    // The actual file type is likely 'FIPA'
+    // with a switch on subtypes (ex: 'FTEX')
+    // but that would be more complicated
+    // to deal with!
+    internal class FIPAFTEX : JUTTexture, IFileAccess
```

Comment:
public?

otherwise it looks good, good work :)

Reply:
hope that I am not too picky. :/

Reply:
@Venomalia - no, you are fine!  I don't know how that class was marked as `internal`, it certainly wasn't me :) ;)

Fixed now!